src/landlab/components/overland_flow/generate_overland_flow_implicit_kinwave.py

```diff
@@ -211,11 +211,15 @@ def __init__(
         ----------
         grid : ModelGrid
             Landlab ModelGrid object
-        runoff_rate : float, optional (defaults to 1 mm/hr)
-            Precipitation rate, mm/hr. The value provided is divided by
-            3600000.0.
-        roughness : float, defaults to 0.01
-            Manning roughness coefficient; units depend on depth_exp.
+        runoff_rate : array_like of float or str (defaults to 1 mm/hr)
+            Precipitation rate, mm/hr. If array, you may either provide
+            a numpy array or the field name str of where the values
+            are stored on the nodes of the ModelGrid.
+        roughness : array_like of float or str (defaults to 0.01)
+            Manning's roughness coefficient(s); units depend on depth_exp.
+            If array, you may either provide a numpy array or the field
+            name str of where the values are stored on the nodes of the
+            ModelGrid.
         changing_topo : boolean, optional (defaults to False)
             Flag indicating whether topography changes between time steps
         depth_exp : float (defaults to 1.5)
```

Comment:
For all of these keyword parameters, it's not needed to include the default values since they are given in the function signature.